nominees/openeyes.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "OpenEyes",
+  "aliases": [
+    "OE"
+  ],
+  "description": "OpenEyes™ is the leading open source electronic patient record (EPR) for ophthalmology. It has been designed by clinicians to be fast in delivering content, fast to navigate and easy (and fun) to use.",
+  "website": "https://openeyes.apperta.org",
+  "license": [
+    {
+      "spdx": "AGPL-3.0",
+      "licenseURL": "https://github.com/AppertaFoundation/openeyes/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceText": "OpenEyes has been developed with the goal of improving ophthalmic outcomes for patients",
+      "evidenceURL": "http://openeyes.apperta.org"
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceText": "OpenEyes promotes innovation through its unique, clinically led open way of working approach, following the Custodian Model",
```

Comment:
I don't think this platform is applicable to any of the targets under SDG 9.

Reply:
Resolved in commit f01582b